### DISCUSSION TITLE
How to change the AppImage icon both in dock and system tray

### DISCUSSION THREAD

**Discussion by aliquo93** [Melvin-Abraham/Google-Assistant-Unofficial-Desktop-Client#494] [General]
I've made a custom launcher using the Siri icon because I use an Apple theme, but when I open the app, it shows in the dock another instance with the original icon and the same icon is in the system tray. How can I edit the AppImage to use my personal icon? Please I need a step-by-step guide!

![1](https://user-images.githubusercontent.com/9071297/130533543-bade1272-5eb8-4070-bb3c-18ed5f7a7a0e.png)

**Comment by aliquo93** [comment]
Anyone who can help, please?

**Reply by Melvin-Abraham** [reply]
@aliquo93 This issue has been raised in #29. This has been added to the backlog for now and will be worked on later...
